deploy/parts/domains-rabbitmq.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -e
+
+echo -n "Prepare RabbitMQ management domain "
```

Comment:
nitpick, I found the plural in the file name a little bit confusing - if I understand it correctly, this script prepares the domain for redis admin, so maybe`domain-rabbit(-admin/management).sh` would be more precise?

Reply:
fixed :)